### DISCUSSION TITLE
Day_03_Public_IP_is_not_accesaable

### DISCUSSION THREAD

**Discussion by nadeemtaj371** [chiche-ds/30-Day-Terraform-challenge-#69] [Day-to-Day Tasks]
Hi everyone,
I hope you guys are doing well, I have done with day_3 tasks but unfortunately, My ec2 machine is not accessible via Public_IP:port_number. Would anyone be able to help?

thanks in advanced

**Comment by chiche-ds** [comment]
check your security groups and verify if the port number is allow

**Reply by nadeemtaj371** [reply]
thank you so much for your quick response. I figured it out, now it's working.
